node/out_response.js

```diff
@@ -34,8 +34,14 @@ function TChannelOutResponse(id, options) {
     self.spanEvent = self.defineEvent('span');
     self.finishEvent = self.defineEvent('finish');
 
+    self.channel = options.channel;
+    self.callingService = options.callingService;
```

Comment:
use an `self.inreq` field instead of `callingService`, `endpoint`, `serviceName`